src/overview/bare-metal.md

```diff
@@ -1,59 +1,53 @@
-[official rust embedded book]: https://docs.rust-embedded.org/
+[embedded-rust-book]: https://docs.rust-embedded.org/
+[embedded-rust-book-no-std]: https://docs.rust-embedded.org/book/intro/no-std.html
 
-## Hardware Abstraction Layers
 
-Previously, the primary focus of `no_std` development was the ESP32 and (to a lesser extent) the ESP8266.
+## Relevant `esp-rs` crates
 
-Now there is a renewed effort to implement `no_std` support for the entire lineup of Espressif devices from the ESP32 and newer. These new HALs can be found in the [esp-rs/esp-hal] repository.
+The table below covers what `no_std` features are supported by Espressif products.
 
-There is also some level of support for Wi-Fi and Bluetooth via [esp-rs/esp-wifi] for ESP32, ESP32-C3, ESP32-S2, and ESP32-S3.
+> **Notes**:
+>
+> - Hover your cursor over the links in the table header to see feature details.
+> - The icon ⏳ indicates that the feature is under development.
+> - Features marked with * must be installed for `no_std` to work.
 
-[esp-rs/esp-hal]: https://github.com/esp-rs/esp-hal
-[esp-rs/esp-wifi]: https://github.com/esp-rs/esp-wifi
+|         | [HAL][esp-rs/esp-hal]* | [PAC][esp-rs/esp-pacs]* | [Wi-Fi][esp-rs/esp-wifi] | [Bluetooth][esp-rs/esp-wifi] | [Heap][esp-rs/esp-alloc] | [Println][esp-rs/esp-println] | [Backtrace][esp-rs/esp-backtrace] | [Storage][esp-rs/esp-storage] |
+| -------- | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+|  ESP32   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-C2 |   ✅   |   ✅   |   ✅   |   ✅   |   ❌   |   ✅   |   ✅   |   ✅   |
+| ESP32-C3 |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-C6 |   ✅   |   ✅   |   ✅   |   ⏳   |   ❌   |   ✅   |   ✅   |   ❌   |
+| ESP32-S2 |   ✅   |   ✅   |   ✅   |   ❌   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-S3 |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-H2 |   ❌   |   ✅   |   ❌   |   ❌   |   ❌   |   ✅   |   ❌   |   ❌   |
+| ESP8266  |   ✅   |   ✅   |   ❌   |   ❌   |   ❌   |   ✅   |   ❌   |   ❌   |
 
-## Chip Support
+The `no_std` features are elaborated on below.
 
-Chip support for `no_std` requires LLVM/Clang support just like for `std`. However, this has no dependency on `esp-idf`. In addition to compiler support, it's necessary to have peripheral access crates (PAC) and hardware abstraction layers (HAL) for your desired chip.
+- **HAL** -- Hardware abstraction layer, see [esp-rs/esp-hal] and [esp-rs/esp8266-hal]
+- **PAC** -- Peripheral access crates, see [esp-rs/esp-pacs]
+- **Wi-Fi** -- Wi-Fi support, see [esp-rs/esp-wifi]
+- **Bluetooth** -- Bluetooth LE support, see [esp-rs/esp-wifi]
+- **Heap** -- Simple heap allocator, see [esp-rs/esp-alloc]
+- **Println** -- `print!`, `println!`, see [esp-rs/esp-println]
+- **Backtrace** -- Exception and panic handlers, see [esp-rs/esp-backtrace]
+- **Storage**  -- Embedded-storage traits to access unencrypted flash memory, see [esp-rs/esp-storage]
```

Comment:
Maybe change the list to be crate centered, eg:
- esp-hal: Hadwared abstraction layers
- esp-wifi: Support for wifi, ble, esp-now...
- ...

Reply:
@SergioGasquez 

Then how about returning to a table format like this? We also use a similar format for `std` crates/repositories.

Should the first column be called repository or feature or crate? Also relevant for `std` crates/repositories.

| Repository | Symbol | Description |
| ------- | :----: | ----------- |
| [esp-rs/esp-hal] | **HAL** | Hardware abstraction layer |
| [esp-rs/esp-pacs] | **PAC** | Peripheral access crates |
| [esp-rs/esp-wifi] | **Wi-Fi** | Wi-Fi support |
| [esp-rs/esp-wifi] | **Bluetooth** | Bluetooth LE support |
| [esp-rs/esp-alloc] | **Heap** | Simple heap allocator |
| [esp-rs/esp-println] | **Println** | `print!`,  `println!` |
| [esp-rs/esp-backtrace] | **Backtrace** | Exception and panic handlers |
| [esp-rs/esp-storage] | **Storage** | Embedded-storage traits to access unencrypted flash memory |

[esp-rs/esp-hal]: https://github.com/esp-rs/esp-hal "Hardware abstraction layer"
[esp-rs/esp8266-hal]: https://github.com/esp-rs/esp8266-hal
[esp-rs/esp-pacs]: https://github.com/esp-rs/esp-pacs "Peripheral access crates"
[esp-rs/esp-wifi]: https://github.com/esp-rs/esp-wifi "Wi-Fi and Bluetooth LE support"
[esp-rs/esp-alloc]: https://github.com/esp-rs/esp-alloc "Simple heap allocator"
[esp-rs/esp-println]: https://github.com/esp-rs/esp-println "print!, println!"
[esp-rs/esp-backtrace]: https://github.com/esp-rs/esp-backtrace "Exception and panic handlers"
[esp-rs/esp-storage]: https://github.com/esp-rs/esp-storage "Embedded-storage traits to access unencrypted flash memory"

Reply:
> Then how about returning to a table format like this? 

I think its okay to have such table, but I would remove the Symbol column

> Should the first column be called repository or feature or crate? 

I would leave the "repository" term because one repo can contain several crates.

Reply:
Decided to keep column **Symbol** and make it first in the table.

If we find a way to make mapping between the table above and this one easier without symbols, we can always delete this column to keep this tables formatting the same as its `std` counterpart.